DCHdigi/README.md

```diff
@@ -0,0 +1,20 @@
+# Drift chamber (DCH) digitizers
+
+## DCHsimpleDigitizer
+
+* Similar to DCHdigi_v01, but it uses the placement matrix of the wires to apply the smearing along/perpendicular to the sense wire
```

Comment:
Since `DCHdigi_v01` is only described  later, one should maybe not described this one based on `DCHdigi_v01` (or reverse the order of the README. A description could be from what is written in the .h doxygen description.